pyproject.toml

```diff
@@ -6,10 +6,10 @@ authors = []
 
 [tool.poetry.dependencies]
 python = "^3.7"
-flask = "^1.1.2"
 python-dotenv = "^0.14.0"
-requests = "^2.26.0"
-pytest = "^7.1.2"
```

Comment:
You seem to have removed pytest?

Run `poetry add pytest` to add it back.

Or better yet, run `poetry add pytest --dev` to add it as a "dev dependency". Then update your Dockerfile to RUN `poetry install --no-dev` on line 11, so that it skips pytest.